controllers/ocicluster_controller_test.go

```diff
@@ -499,7 +499,18 @@ func TestOCIClusterReconciler_reconcileDelete(t *testing.T) {
 }
 
 func getOciClusterWithNoOwner() *infrastructurev1beta1.OCICluster {
-	ociCluster := getOCICluster()
+	ociCluster := &infrastructurev1beta1.OCICluster{
```

Comment:
The method was calling into machine reconciler test code, which I had changed. We should not have dependency on machine reconciler test.